virtcontainers/kata_agent.go

```diff
@@ -570,8 +570,6 @@ func constraintGRPCSpec(grpcSpec *grpc.Spec) {
 			grpcSpec.Mounts[idx].Type = "tmpfs"
 			grpcSpec.Mounts[idx].Source = "shm"
 			grpcSpec.Mounts[idx].Options = []string{"noexec", "nosuid", "nodev", "mode=1777", "size=65536k"}
-
-			break
```

Comment:
I don't understand how this change relates to bind mounts?

Reply:
Oh yeah this one was a slightly different fix that is also needed. Let me split that in a different commit.